src/features/user/Settings/EditProfile/AddressManagment/AddressManagement.module.scss

```diff
@@ -97,3 +93,61 @@
 .addressActions li:not(:last-child) {
   border-bottom: 1px solid rgba(189, 189, 189, 1);
 }
+// address action form
+
+.addressFormContainer {
+  display: flex;
+  flex-direction: column;
+  gap: 40px;
+  min-width: 565px;
+  background: rgba(255, 255, 255, 1);
+  border-radius: 16px;
+  padding: 20px;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  h2 {
+    font-weight: 700;
+    font-size: $fontSixteen;
+  }
+}
+
+.addressForm {
+  display: flex;
+  flex-direction: column;
+  gap: 23px;
+}
+
+.formButtonContainer {
+  display: flex;
+  gap: 8px;
+  justify-content: space-between;
+  align-items: end;
+  .addAddressButton,
+  .cancelButton {
+    min-width: 252px;
+  }
```

Comment:
As per the design, there should be more space between the last line of the form fields (state/country) and the buttons.